mifosng-android/src/main/java/com/mifos/services/data/CenterPayload.java

```diff
@@ -4,18 +4,63 @@
  */
 package com.mifos.services.data;
 
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import com.mifos.api.local.MifosBaseModel;
+import com.mifos.api.local.MifosDatabase;
+import com.raizlabs.android.dbflow.annotation.Column;
+import com.raizlabs.android.dbflow.annotation.ModelContainer;
+import com.raizlabs.android.dbflow.annotation.PrimaryKey;
+import com.raizlabs.android.dbflow.annotation.Table;
+
 /**
  * Created by nellyk on 1/22/2016.
  */
-public class CenterPayload {
+@Table(database = MifosDatabase.class)
```

Comment:
@mayank-kgp We already published the app on play store, so you have to migrate database while creating new table in the database.
 Here is guide of DBFlow migration database https://github.com/agrosner/DBFlowDocs/tree/99495e3566e38785870f6c580caa3915709f8d1f